tests/test_utils.py

```diff
@@ -138,7 +130,7 @@ def test_get_west_most_point():
             'coordinates': [
                 [
                     [-122.201872, 41.1562811],
-                    [-122.201872, 39.2490859],
+                    [-122.201872, 3.240859],
```

Comment:
What's the context behind this change?

Reply:
Don't know... I think I'm going to blame the cat on this one.